changelogs/unreleased/4332-service-details-bugs.yml

```diff
@@ -0,0 +1,8 @@
+change-type: patch
+description: 'Fix service callbacks issues in display, in readabilty and form behaviour after adding component'
```

Comment:
after adding a Callback
(Components are a concept that we understand, but for end users reading the changelogs, it doesn't make a lot of sense)